inst/htmlwidgets/cytoscapeNetwork.js

```
@@ -0,0 +1,484 @@
+        /* Right panel — shared background for button + legend */
+        var PANEL_BG = "#f8f9fa";
+        var rightPanel = document.createElement("div");
+        rightPanel.style.cssText = [
+          "width:" + PANEL_W + "px",,
+          "flex-shrink:0",
+          "display:flex",
+          "flex-direction:column",
+          "background:" + PANEL_BG,
+          "border-left:1px solid #dee2e6",
+          "box-sizing:border-box"
+        ].join(";");
```

Biome (2.4.4)

[error] This array contains an empty slots..

(lint/suspicious/noSparseArray)